apps/dashboard/src/app/App.tsx

```diff
@@ -4,6 +4,7 @@
 import { ITheme } from "@fluentui/react";
 import { generateRoute } from "@responsible-ai/core-ui";
 import { Language } from "@responsible-ai/localization";
```

Comment:
do you think it might be useful to add a small blurb about this in the contributing.md file:
https://github.com/microsoft/responsible-ai-toolbox/blob/main/CONTRIBUTING.md
to tell other developers how they could leverage this flight flag?

Reply:
I think that's a good idea. This knowledge shouldn't just get lost.